features/facebookImport/src/views/explore/details.jsx

```diff
@@ -15,10 +15,10 @@ const ExploreDetails = () => {
         mode: Story.MODES.DETAILS,
     });
     return (
-        <div className="details-view">
-            <h1>{activeStory.title}</h1>
+        <Screen className="details-view" layout="poly-standard-layout">
+            <h2>{activeStory.title}</h2>
```

Comment:
Right, you're changing h2 → h1. But leaving the same size?

Reply:
If you look at the size it's pretty small for an h1, so I thought makes more sense to be h2

Reply:
Went back to h1, as you said it's the biggest font size we have right now